### PR TITLE
add pdm install to repository

### PR DESCRIPTION
**Pull request by tp-peacock**
Install via PDM and pre-commit

review @rhjohnstone

### CODE REVIEW

File: .pdm.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[venv]
+with_pip = true
+
+[python]
+path = "/Users/peacockthomas/sg/results-app/.venv/bin/python"
```
Comment (rhjohnstone) [inline .pdm.toml:5]: Can we either omit this line from the committed file, or something?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+unfixable = ["B"]
+
+[tool.ruff.per-file-ignores]
+"__init__.py" = ["E402", "F401"]
+
+[tool.ruff.mccabe]
+max-complexity = 5
+
+[tool.pytype]
+inputs = ["evaluators"]
```
[the remark this answers is not in the document]
Comment (tp-peacock) [reply pyproject.toml]: oops

(I removed it)

**Review by rhjohnstone — COMMENTED** [review]
Two small questions as comments

**Review by rhjohnstone — APPROVED** [review]

File: .pdm.toml (hunk 1 above)
Comment (tp-peacock) [reply .pdm.toml]: Removed!

**Review by rhjohnstone — APPROVED** [review]

**Pull request merged**